site/glossary.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="author" content="hsianglingT">
+  <meta name="description" content="This is a website for learning HTML/CSS/Javascript">
+  <link rel="icon" type="image/png" href="images/favicon-96x96.png" sizes="96x96">
```

Comment:
If you pop this into the real favicon generator site you should be able to get a `.ico` file which would allow you to include:
```suggestion
  <link rel="icon" type="image/png" href="images/favicon-96x96.png" sizes="96x96">
  <link rel="icon" type="image/x-icon" href="images/favicon.ico">
```